docs/src/add-services/_index.md

```diff
@@ -121,6 +121,13 @@ Add them to the `type` key of the [service configuration](#1-configure-the-servi
 <!-- To update the versions in this table, use docs/data/registry.json -->
 {{< readFile file="src/registry/images/tables/services_supported.md" markdownify="true">}}
 
+### Service versions
+
+These services generally follow [semantic versioning conventions](https://semver.org/).
+You choose which major/minor version to add to your project.
+Patch versions are applied periodically for bug fixes and the like.
```

Comment:
If the idea is to be explicit about what users can do, the first part probably shouldn't be passive.

Reply:
But that's why the previous sentence used "You" -- that's the part they can handle themselves.

Reply:
Since the PR adds clarification about the process, the "limitations" and helps clarify what the customer can (and can't change). I'd say we could dismiss the comments for now (unless we come up with a simpler explanation), since it might be a bit over the original scope of that PR?